caproto/threading/client.py

```diff
@@ -893,8 +893,7 @@ def send(self, *commands):
         # be send, and convert them to buffers.
         buffers_to_send = self.circuit.send(*commands)
         # Send bytes over the wire using some caproto utilities.
-        # ca.send_all(buffers_to_send, self._socket_send)
-        self.socket.sendall(b''.join(buffers_to_send))
+        ca.send_all(buffers_to_send, self._socket_send)
```

Comment:
Phew. I read this original change and was like "Whaaaaa? I understand nothing anymore...."